tests/memgraph/test_query_builder.py

```diff
@@ -593,7 +593,61 @@ def test_add_string_complete(memgraph):
     mock.assert_called_with(expected_query)
 
 
-def test_node_object(memgraph):
+def test_node_instance(memgraph):
+    class User(Node):
```

Comment:
Can we promote these in-test defined classes to fixtures, and use them somehow as parameters for the tests?